packages/ozone/src/db/migrations/20250415T201720309Z-verification.ts

```diff
@@ -0,0 +1,25 @@
+import { Kysely, sql } from 'kysely'
+
+export async function up(db: Kysely<unknown>): Promise<void> {
+  await db.schema
+    .createTable('verification')
+    .addColumn('uri', 'text', (col) => col.primaryKey())
```

Comment:
Since we don't keep the full record contents around, keeping around the `cid` in here would be a nice backstop.

Reply:
Over time the same record may have many CIDs but at a given time it just has one, so we wouldn't need to make it a composite key.